### ISSUE TITLE
Custom Ladspa Plugins

### ISSUE THREAD

**Issue by MalexTP** [wwmm/easyeffects#1156]
Is it possible to use custom Ladspa plugins? I want to use LoudMax but following the plugin's installation steps it isn't appearing in EasyEffects plugin list

**Comment by wwmm** [comment]
No. It isn't possible to use any custom plugin. The reason is that EasyEffects(or PulseEffects) were never meant to be a general purpose plugin host like Ardour or Carla. So doing that now is hard.

**Comment by Digitalone1** [comment]
@MalexTP you can do it with Carla and pw-jack. There a guide on Pipewire article in Arch Linux documentation.

**Comment by bangbangsheshotmedown** [comment]
Any plan to revisit that, or add an exception for LoudMax?

https://loudmax.blogspot.com/2023/?m=1

It's very effective and i'd like to be able to use it with easyeffects

**Comment by Digitalone1** [comment]
> Any plan to revisit that, or add an exception for LoudMax?
> 
> https://loudmax.blogspot.com/2023/?m=1
> 
> It's very effective and i'd like to be able to use it with easyeffects

Did you try the Maximizer?
